.ci/upload-build.sh

```diff
@@ -20,14 +20,22 @@ file_content_type="application/octet-stream"
 files_to_upload=( dist/* )
 
 for fpath in "${files_to_upload[@]}"; do
+	echo
 	echo "Uploading '$fpath' to Github..."
 	name=$(basename "$fpath")
 	url_to_upload="https://uploads.github.com/repos/$CIRRUS_REPO_FULL_NAME/releases/$CIRRUS_RELEASE/assets?name=$name"
-	curl -X POST \
+	curl -i -X POST \
 		--data-binary @$fpath \
 		--header "Authorization: token $GITHUB_TOKEN" \
 		--header "Content-Type: $file_content_type" \
 		$url_to_upload
+	echo
```

Comment:
Please remove that `echo`, we are already inserting a linebreak at the top of the loop or right below.

Reply:
This echo is for the curl. The one above is for a stylistic line break so that the various parts are visually broken up.

It's called ux.

Reply:
"This line deliberately left blank"

Reply:
> It's called ux

No need for snark...
It was entirely unobvious that `curl` fails to insert a linebreak before returning.